### ISSUE TITLE
audio file not found

### ISSUE THREAD

**Issue by modpotato** [rogermiranda1000/TwitchTTSBot#6]
FileNotFoundError: [Errno 2] No such file or directory: 'D:\\ttsbot\\TwitchTTSBot\\TwitchTTSBot\\..\\audio-server\\audios\\6e3ec4a9765040b6a98c143bf068c9ad.wav'

 doesnt seem to be contacting the rvc server, no idea why
inference from the webui is fine

D:.
├───audio-server
│   ├───audios
│   └───__pycache__
├───rvc-tts-webui
│   ├───assets
│   ├───lib
│   │   └───infer_pack
│   │       ├───modules
│   │       │   └───F0Predictor
│   │       └───__pycache__
│   ├───weights
│   │   └───jayuu
│   └───__pycache__
├───services
└───TwitchTTSBot
    ├───audios
    ├───commands
    ├───configs
    │   └───command_configs
    ├───models
    ├───mods
    ├───processing
    │   └───__pycache__
    ├───synthesizers
    │   └───__pycache__
    └───__pycache__

 pip freeze
aiofiles==23.2.1
aiohappyeyeballs==2.3.4
aiohttp==3.10.0
aiosignal==1.3.1
altair==5.3.0
annotated-types==0.7.0
antlr4-python3-runtime==4.8
anyio==4.4.0
async-timeout==4.0.3
attrs==23.2.0
audioread==3.0.1
bidict==0.23.1
bitarray==2.9.2
blinker==1.8.2
Brotli @ file:///C:/b/abs_3d36mno480/croot/brotli-split_1714483178642/work
certifi @ file:///C:/b/abs_7bbxxprxir/croot/certifi_1720453609121/work/certifi
cffi==1.16.0
charset-normalizer @ file:///croot/charset-normalizer_1721748349566/work
click==8.1.7
colorama==0.4.6
contourpy==1.2.1
cycler==0.12.1
Cython==3.0.10
decorator==5.1.1
discum @ git+https://github.com/Merubokkusu/Discord-S.C.U.M.git@3a3d9334f66e2f7c7954fcb616a06c52b252804e
dnspython==2.6.1
edge-tts==6.1.7
email_validator==2.2.0
exceptiongroup==1.2.2
fairseq==0.12.2
faiss-cpu==1.7.4
fastapi==0.111.1
fastapi-cli==0.0.4
ffmpy==0.4.0
filelock @ file:///C:/b/abs_f2gie28u58/croot/filelock_1700591233643/work
filetype==1.2.0
Flask==3.0.3
Flask-SocketIO==4.3.1
fonttools==4.53.1
frozenlist==1.4.1
fsspec==2024.6.1
gmpy2 @ file:///C:/ci/gmpy2_1645455782955/work
gradio==3.38.0
gradio_client==0.5.1
greenlet==3.0.3
h11==0.14.0
httpcore==1.0.5
httptools==0.6.1
httpx==0.27.0
huggingface-hub==0.24.5
hydra-core==1.0.7
idna @ file:///C:/b/abs_aad84bnnw5/croot/idna_1714398896795/work
importlib_metadata==8.2.0
importlib_resources==6.4.0
itsdangerous==2.2.0
Jinja2 @ file:///C:/b/abs_92fccttino/croot/jinja2_1716993447201/work
joblib==1.4.2
jsonschema==4.23.0
jsonschema-specifications==2023.12.1
kiwisolver==1.4.5
librosa==0.9.1
linkify-it-py==2.0.3
llvmlite==0.43.0
lxml==5.2.2
markdown-it-py==2.2.0
MarkupSafe @ file:///C:/b/abs_ecfdqh67b_/croot/markupsafe_1704206030535/work
matplotlib==3.9.1
mdit-py-plugins==0.3.3
mdurl==0.1.2
mkl-fft @ file:///C:/b/abs_19i1y8ykas/croot/mkl_fft_1695058226480/work
mkl-random @ file:///C:/b/abs_edwkj1_o69/croot/mkl_random_1695059866750/work
mkl-service==2.4.0
mpmath @ file:///C:/b/abs_7833jrbiox/croot/mpmath_1690848321154/work
multidict==6.0.5
netifaces==0.10.6
networkx @ file:///C:/b/abs_3bxnu56g9d/croot/networkx_1717597507456/work
numba==0.60.0
numpy==1.23.5
omegaconf==2.0.6
orjson==3.10.6
packaging==24.1
pandas==2.2.2
pillow @ file:///C:/b/abs_32o8er3uqp/croot/pillow_1721059447598/work
platformdirs==4.2.2
pooch==1.8.2
portalocker==2.10.1
praat-parselmouth==0.4.3
pycparser==2.22
pydantic==2.8.2
pydantic_core==2.20.1
pydub==0.25.1
Pygments==2.18.0
pyparsing==3.1.2
pypeln==0.4.9
PySocks @ file:///C:/ci/pysocks_1605307512533/work
python-dateutil==2.9.0.post0
python-dotenv==1.0.1
python-engineio==3.13.2
python-multipart==0.0.9
python-socketio==4.6.0
PythonTwitchBotFramework==2.11.5
pytz==2024.1
pywin32==306
pyworld==0.3.4
PyYAML==6.0.1
referencing==0.35.1
regex==2024.7.24
requests @ file:///C:/b/abs_9frifg92q2/croot/requests_1721410901096/work
requests-toolbelt==1.0.0
resampy==0.4.3
rich==13.7.1
rpds-py==0.19.1
sacrebleu==2.4.2
scikit-learn==1.5.1
scipy==1.13.1
semantic-version==2.10.0
shellingham==1.5.4
simple-websocket==1.0.0
six==1.16.0
sniffio==1.3.1
soundfile==0.12.1
SQLAlchemy==2.0.31
starlette==0.37.2
stopit==1.1.2
sympy @ file:///C:/b/abs_82njkonm7f/croot/sympy_1701397685028/work
tabulate==0.9.0
threadpoolctl==3.5.0
toolz==0.12.1
torch==2.0.0
torchaudio==2.0.0
torchcrepe==0.0.20
torchvision==0.15.0
tqdm==4.66.4
typer==0.12.3
typing_extensions @ file:///C:/b/abs_0as9mdbkfl/croot/typing_extensions_1715268906610/work
tzdata==2024.1
ua-parser==0.18.0
uc-micro-py==1.0.3
urllib3==1.26.18
uvicorn==0.30.4
watchfiles==0.22.0
websocket-client==0.59.0
websockets==11.0.3
Werkzeug==3.0.3
win-inet-pton @ file:///C:/ci/win_inet_pton_1605306162074/work
wsproto==1.2.0
yarl==1.9.4
zipp==3.19.2

**Comment by miranda1000** [comment]
@modpotato I'd say open the `rvc-tts-webui` website and try to manually synthetize a text. If there's any error, there it should tell you

**Comment by modpotato** [comment]
yeah no luck so far, the webui works perfectly fine but its like the actual bot isnt calling out to the api (no logs of it in the rvc screen) and it just checks for a file to play which doesnt exist in the first place

**Comment by miranda1000** [comment]
Are you using linux? Last guy that tried to ru  this on Windows didn't succeed...

You can use WSL, but not native Windows

**Comment by modpotato** [comment]
im not sure how it would be a windows issue, though it could by a python os library issue idk what else

**Comment by miranda1000** [comment]
> im not sure how it would be a windows issue

Someone asked me on DMs and I don't remember it fully, but it was a combination of path concatenation (that it seems to be your issue), and once I solved that then there was an issue with gradio_client (but I don't remember)

**Comment by miranda1000** [comment]
Actually windows path were fixed on https://github.com/rogermiranda1000/TwitchTTSBot/commit/cca197167308debeac7c419f6e02a3009703e793 so I don't know what could be happening now

**Comment by modpotato** [comment]
i will say i didnt have the audio server running which is probably why now thtat i look anymore
as the bind port is the same as the rvc webui, what ports are bound to what on your configuration?

**Comment by miranda1000** [comment]
You have to run the server on background. Please, check the README

**Comment by modpotato** [comment]
when i start the bot a webserver starts along side it which populates said port
